fiftyone/core/utils.py

```diff
@@ -662,8 +666,12 @@ def load_xml_as_json_dict(xml_path):
     Returns:
         a JSON dict
     """
-    with open(xml_path, "rb") as f:
-        return xmltodict.parse(f.read())
+    try:
+        with open(xml_path, "rb") as f:
+            return xmltodict.parse(f.read())
+    except ExpatError as ex:
+        logger.error(f"Failed to read {xml_path}: {ex}")
```

Comment:
nit: per [style guide](https://github.com/voxel51/fiftyone/blob/develop/STYLE_GUIDE.md#logging) we currently prefer directly raising exceptions to `logger.error()`:

```py
raise ExpatError(f"Failed to read {xml_path}: {e}")
```